docs/docs/installation/docker.mdx

```diff
@@ -61,7 +64,20 @@ build times, larger images, lower layer cache hit rate, ...).
 For production use cases, we recommend that you derive our `lean` image(s) and
 add database support for the database you need.
 
-## On ARM builds and working with Apple silicon
+## On supporting arm64 AND amd64
+
+Only the release builds are multi-platform, supporting `linux/arm64`
```

Comment:
Based on all the github noise, I'm just wondering if there are _currently_ versions with this support, or if we're lacking this support until the _next_ version. I'm guessing the former, and not the latter 🤞